### DISCUSSION TITLE
Picklist overview in a TM1 model

### DISCUSSION THREAD

**Discussion by wimgielis** [cubewise-code/tm1py#673] [Ideas]
Hello,

I wrote a generic TI process to create an overview of picklists in a TM1 model.
The output is 2 text files: picklists created with a text attribute, and picklists created with a real picklist cube.

If anyone wants to 'translate' it in tm1py code, you can :-) Please keep my name in the header of the script.
Also, comments on the TI process itself are welcome too.

For example, parsing rules in a }Picklist cube might be an interesting task, if it hasn't been done before. Parsing rules in general in tm1py maybe hasn't been done before, no idea to be honest.

[TECH_picklist_overview.zip](https://github.com/cubewise-code/tm1py/files/7912982/TECH_picklist_overview.zip)

Best regards,

Wim

**Comment by wimgielis** [comment]
Posted at TM1forum as well:
http://www.tm1forum.com/viewtopic.php?t=16101

**Comment by MariusWirtz** [comment]
Thanks for the contribution! Will check it out when I have a quiet moment :)
